be/src/io/cache/block_file_cache.cpp

```diff
@@ -393,6 +393,9 @@ FileBlocks BlockFileCache::get_impl(const UInt128Wrapper& hash, const CacheConte
 
     auto& file_blocks = it->second;
     DCHECK(!file_blocks.empty());
+    if (file_blocks.empty()) {
```

Comment:
should we remove the unexpected key?
and we have need to print some waring for this exceptional branch, e.g., hex(key), path and etc.